### PR TITLE
To Do list: list structure

### PR DESCRIPTION
**Pull request by VSeray**
In this feature, I will be applying the following functionality.

Set up a new project with webpack that is based on the webpack exercise I have already completed.
Created an index.html file and an empty To-Do List placeholder (
or
element). The index.html file is set as a template using the HTML Webpack Plugin.
Created an index.js file and set an array of some simple to-do tasks (array of objects).
Written a function to iterate over the tasks array and populate an HTML list item element for each task.
On page load rendered dynamically creating a list of tasks in the dedicated placeholder.
Apply style for the to-do list.
Update Html with ul element and semantic UI CDN.

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -0,0 +1,5 @@
+{
+  "devDependencies": {
+    "webpack-cli": "^4.9.1"
+  }
+}
```
Comment (TedLivist) [inline package.json]: - [x] I am unable to build (or bundle) your app. Your project is missing the required dependencies and Webpack commands. You only have one dependency in your package file. Generally, if you set up your project correctly using the lessons as provided in the curriculum, this package file should be way more populated than this and I should be able to run predefined webpack commands like `npm run build` and/or `npm start`. Webpack devserver should be set up. Kindly refer to these lessons **[Webpack](https://github.com/microverseinc/curriculum-javascript/blob/main/todo-list/lessons/lesson_webpack.md)** and **[Set up project with webpack](https://github.com/microverseinc/curriculum-javascript/blob/main/todo-list/exercises/exercise_set_up_project_with_webpack.md)** (as provided in your **Progress** tab) for a better understanding of the project setup with Webpack.

  There seems to be no relationship between the `src/index.js` file and the `dist/main.js` file. There is a **"Hello webpack"** task in the `main.js` file that does not exist anywhere in your `index.js` file. Kindly refer to the above lessons and set up your project with Webpack correctly.

File: README.md [hunk 2]
```diff
@@ -1 +1,66 @@
-# To-Do-list
\ No newline at end of file
+# Awesome-books
```
Comment (TedLivist) [inline README.md]: - [ ] You are building the To-Do list project but have a title for Awesome Books. Kindly correct this.

File: README.md [hunk 3]
```diff
@@ -1 +1,66 @@
...
+Please get your browser update. To get a local copy up and running follow these simple steps.
+
+Run the command below from the Terminal:
+
+navigate to your projects folder
+
+run git clone (https://github.com/VSeray/Portfolio) in your terminals
+
+run cd microverse_portfolio
+
```
Comment (TedLivist) [inline README.md]: - [x] You have commands for the HTML portfolio project whereas you are building a different project. Kindly fix this by replacing them with the correct link and text

File: README.md [hunk 4]
```diff
@@ -1 +1,66 @@
...
+- LinkedIn: [Linkdin](https://www.linkedin.com/in/vana-seraydarian-936687191/?lipi=urn%3Ali%3Apage%3Ad_flagship3_feed%3BNyso4dw6Tz6UBL%2Fqkjvtvw%3D%3D)
+
+👤 Author
+
+- GitHub: [@Anny85-code](https://github.com/Anny85-code)
+
+- Twitter: @Annyudo8
+
+- LinkedIn: [LinkedIn](https://www.linkedin.com/in/aniekan-udo-665b65213/)
```
Comment (TedLivist) [inline README.md]: - [x] I believe this project is solo. Kindly remove this second author for proper documentation.

File: src/index.js [hunk 5]
```diff
@@ -0,0 +1,103 @@
+import './style.css';
+
+let ToDoArray = JSON.parse(localStorage.getItem('ToDoArray') || '[]');
```
Comment (TedLivist) [inline src/index.js]: - [ ] As listed in the requirements there should be an array containing some tasks with each task having the following properties: description [`string`], completed [`bool`], index: [`number`]. Ensure you add this.
- [ ] Also ensure that this array of tasks is rendered dynamically to the DOM on page load.

**Review by TedLivist — CHANGES_REQUESTED** [review]
## STATUS: CHANGES REQUIRED ♻️ ♻️

Hi @VSeray,

Good job so far!
✔️ Linters are set up correctly
However, there are some issues that you still need to work on to go to the next project but you are almost there!

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by elyor-doniyorov — APPROVED** [review]
Hi @VSeray, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**